### PR TITLE
CONCF-470 Video uploading fix

### PR DESCRIPTION
**Pull request by dianafa**
**TLDR;** The failed video upload issue was caused by wgIsGhostVideo which was set in [VideoInfoHooksHelper.class.php](https://gist.github.com/dianafa/898d5e2186d247d78162#file-videoinfohookshelper-class-php-L329) and was causing the error while uploading video.

At first try, before the video was not yet in the DB it set wgIsGhostVideo as true and thus after, when actually everything went correctly, onCheckGhostFile file was set to null and there was a failure. After the page reload everything appeared correctly (it was correctly in fact, only the error caused by wgIsGhostVideo check was misleading).

Function wfFindFile( $videoTitle ); was actually a wrapper for RepoGroup::singleton()->findFile( $videoTitle ); which contained unuseful in case of video uploading wgIsGhostVideo check and caused error.

ping: @bognix @vforge

### CODE REVIEW

**Comment by vforge** [comment]
LGTM :heavy_check_mark:

**Comment by bognix** [comment]
ghost video is a video from www.video.wikia.com which doesn't have a record on local wiki. Please make sure while testing that uploading this videos still works and they are playable.

**Comment by bognix** [comment]
LGTM
